po/ko.po

```diff
@@ -11471,14 +11493,19 @@ msgid ""
 "com/chromiumos/bazel/), and [Fuchsia](https://source.android.com/docs/setup/"
 "build/bazel/introduction) to Bazel."
 msgstr ""
+"[Android](https://source.android.com/docs/setup/build/bazel/introduction), "
+"[ChromeOS](https://chromium.googlesource.com/chromiumos/bazel/), [Fuchsia]"
+"(https://source.android.com/docs/setup/build/bazel/introduction)를 Bazel로 전"
+"환할 계획입니다."
```

Comment:
Android stopped the transition to Bazel a few months ago. We may have to amend the original text.

Reply:
Created an issue to track this.